grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -65,6 +65,7 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
         "hasPrepaymentInvoice",
         "hasRegularInvoice",
         "disableRefresh",
+        "amountInvoiced",
```

Comment:
Do you still have a transient for this? Is this potentially for the same thing as `getInvoicedAmount`?

Reply:
don't need it, removing it now